docs/hf_hub.rst

```diff
@@ -0,0 +1,72 @@
+.. _hf_hub:
+
+scikit-learn Models on HuggingFace Hub
+======================================
+
+This library allows you to initialize and create a model repository compatible
+with HuggingFace Hub, which you can push to the hub and call an API to get the
```

Comment:
I think you could also mention other benefits apart from the API
* A widget to try the model directly in the browser
* Metadata tags for discoverability
* Collaborating with others in a model
* Downloading/persisting in a way easy to share